spec/models/base_decorator_spec.rb

```diff
@@ -0,0 +1,10 @@
+require_relative '../../base_decorator'
```

Comment:
- [ ] Hey there! 🚀 I've observed an issue with the relative path you're using. It appears you're referencing "base_decorator" at the root of your project, but the actual path should be "decorator." 😬 Kindly double-check this and make the necessary correction to ensure everything works smoothly. 🛠️ Your attention to this matter is greatly appreciated! 🙏

```suggestion
require_relative '../../decorator'
```

![image](https://github.com/CVILLA09/OOP-School-Library/assets/112261237/64170422-5cc3-4a41-b816-c86b57fe96fe)